packages/php-wasm/compile/build-assets/Makefile

```diff
@@ -0,0 +1,221 @@
+PKG_CONFIG_PATH=/root/lib/lib/pkgconfig
+TIMER=(which pv > /dev/null && pv --name '${@}' || cat)
+
+.PHONY: all
+all: build
+
+.PHONY: build
+build: libz.a libzip.a libpng.a libxml2.a libsqlite3.a libbison.a
+	$(TIMER) && echo "Done building"
+
+.PHONY: clean
+clean:
+	docker rmi my-image
+
+.PHONY: run
+run:
+	docker run my-image
+
+.PHONY: shell
+shell:
+	docker run -it my-image /bin/bash
+
+libz.a:
+	chmod a+x zlib/configure;
+	cd zlib && emconfigure ./configure --prefix=/root/lib
+	cd zlib && (EMCC_FLAGS="-fPIC" emmake make || true)
+	cd zlib && EMCC_FLAGS="-fPIC" emmake make install
+	touch libz.a
+
+# @TODO: Use 1.2.0 for PHP <= 7.3 and 1.9.2 versions for PHP > 7.4
+libzip_old:
+	curl -k https://libzip.org/download/libzip-1.2.0.tar.gz -o libzip.tar.gz && \
+	tar -xzf libzip.tar.gz && \
+	mv libzip-1.2.0 libzip_old && \
+	mkdir -p libzip_old/build
+
+libzip.a: libz.a libzip_old
+	cd libzip_old/build && \
+	emcmake cmake \
+		-DCMAKE_INSTALL_PREFIX=/root/lib \
+		-DZLIB_LIBRARY=/root/lib/lib/libz.a \
+		-DZLIB_INCLUDE_DIR=/root/lib/include \
+		.. && \
+	EMCC_SKIP="-lz" EMCC_FLAGS=" -sSIDE_MODULE " emmake make && \
+	EMCC_SKIP="-lz" EMCC_FLAGS=" -sSIDE_MODULE " emmake make install && \
+	touch libzip.a
+
+libxml2:
+	env GIT_SSL_NO_VERIFY=true git clone https://gitlab.gnome.org/GNOME/libxml2.git libxml2 \
```

Comment:
can't pass an opportunity to callout `GIT_SSL_NO_VERIFY=true` in an official project and ask if we really need to do this? who wants to be the vector through which someone's computer is attacked? 🙌